src/main/java/com/rpgrealm/rpgrealm/models/User.java

```diff
@@ -0,0 +1,81 @@
+package com.rpgrealm.rpgrealm.models;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
```

Comment:
Let's go ahead and set up the correct tables for our users. It will be easier I think to have the database right before we push it up to the server.